docs/wiki/Self-Hosted.md

```diff
@@ -0,0 +1,56 @@
+# Self-hosted agents/runners
+
+The AzOps pipelines/actions are running on Microsoft-hosted agents or GitHub Actions hosted virtual environments by default. Depending on your organizations security requirements you might want to use self-hosted agents/runners instead. AzOps have full support for the use of self-hosted agents/runners and this article outlines the requirements needed.
```

Comment:
Can we perhaps expand with one or two more sentences what the additional benefits are, such as managed identity support etc?